app/components/Values.tsx

```diff
@@ -10,17 +12,31 @@ export default function Values() {
   return (
     <div className="flex flex-col">
       <div className="flex flex-col md:flex-row md:items-end">
-        <div className="flex">
-          <Image
-            src={singingFrogWithCloudDesktop}
-            alt="Frog with a crown singing a binary song"
-            className="flex-1 p-4 max-md:hidden"
-          />
-          <Image
-            src={singingFrogWithCloudMobile}
-            alt="Frog with a crown singing a binary song"
-            className="flex-1 p-4 md:hidden"
-          />
+        <div className="flex flex-col">
+          <div className="flex">
+            <Image
+              src={rainMovingCloud}
+              alt="A rainy cloud"
+              className="flex-1 animate-slide-in p-4 max-md:hidden"
+            />
+            <Image
+              src={rainMovingCloud}
+              alt="A rainy cloud"
+              className="flex-1 animate-slide-in p-4 md:hidden"
+            />
+          </div>
+          <div className="flex">
+            <Image
+              src={singingFrog}
+              alt="Frog with a crown singing a binary song"
+              className="flex-1 p-4 max-md:hidden"
+            />
+            <Image
+              src={singingFrog}
+              alt="Frog with a crown singing a binary song"
+              className="flex-1 p-4 md:hidden"
+            />
+          </div>
```

Comment:
It's weird that these images are duplicated with _exactly_ the same CSS... Why? 🤔

Reply:
OK it was duplicating between twice the same _exact_ image but one with `max-md:hidden` and one with `md:hidden` 🙃 
I remove it here e059781b42d2759f64b649d236e2930810599979